spec/controllers/dashboard_controller_spec.rb

```diff
@@ -1,3 +1,17 @@
 require "rails_helper"
 RSpec.describe DashboardController, type: :controller do
+  it "hits dashboard controller" do
+    session[:bikes] = [{name: "Bike", distance: 1000}]
+    session[:clubs] = []
+    session[:ftp] = 200
+    session[:member_since] = "2011"
+    user = User.create(first_name: "Mike", token: ENV['STRAVA_PERSONAL'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.